xaitk_saliency_demo/app/core.py

```diff
@@ -168,6 +172,7 @@ def update_model_execution(self):
 
         # classes
         classes = results.get("classes", [])
+        self.state.classes_pickle = pickle.dumps(classes)
```

Comment:
not sure this needs to go to the UI?

Reply:
Could use `server.context` if you need a central place to put things that stay on the Python side. But maybe that could also be on the `self.something`?

Reply:
I moved to `self.classes` instead of forcing the dictionary into `self.state`.